docs/rules/README.md

```diff
@@ -138,8 +137,8 @@ These rules are specific to JavaScript running on Node.js or using CommonJS in t
 * [no-mixed-requires](no-mixed-requires.md) - disallow mixing regular variable and require declarations
 * [no-new-require](no-new-require.md) - disallow use of `new` operator with the `require` function
 * [no-path-concat](no-path-concat.md) - disallow string concatenation with `__dirname` and `__filename`
+* [no-process-env](no-process-env.md) - disallow use of `process.env`
```

Comment:
Ouch, I would probably assume that this rule did not exist if I did not find it under Node.js